spec/tracebacks_spec.lua

```diff
@@ -6,12 +6,12 @@
 local util = require "spec.util"
 
 local function assert_test(example, expected_content)
-    local cdir = util.shell_quote("spec/tracebacks/"..example)
-    local ok, err = util.execute(string.format("cd %s && make", cdir))
-    assert(ok, err)
+    assert(util.execute("make --quiet tests"))
 
+    local cdir  = util.shell_quote("spec/tracebacks/"..example)
+    local ptrun = util.shell_quote("../../../src/pt-run")
```

Comment:
Wouldn't it be easier to just install Pallene Tracer and run the tests? As we would be using extra lines in the CI script to atleast compile `pt-run` to run this test anyway, wouldn't it make sense to properly install it first, as it would also test proper installation?

Reply:
The traceback ".so" files are always compiled with the local version of ptracer. Therefore, I felt it was more consistent to also run the pt-run with the local version; it would be misleading to run system pt-run with local tracebacks.

But maybe we can set things up so that the CI tests a full install? In any case, I would rather leave that to a future PR.

Reply:
The alternative would be always use the system ptracer when compiling. (As we were doing before; it installed the ptracer before compiling). However I really don't like that because it needs sudo to compile, and is not what people expect from a makefile.

Reply:
I presume it does make sense, you shouldn't have to install when you are testing stuff.